spec/rdiff-backup-test/nrpe_false_create_server_spec.rb

```diff
@@ -0,0 +1,161 @@
+require_relative '../spec_helper'
+
+describe 'rdiff-backup-test::nrpe_false_create_server' do
+  [CENTOS_6_OPTS, CENTOS_7_OPTS].each do |pltfrm|
+    context "on #{pltfrm[:platform]} #{pltfrm[:version]}" do
+      let(:runner) do
+        ChefSpec::SoloRunner.new(
+          pltfrm.dup.merge(step_into: ['rdiff-backup'])
+        )
+      end
+      cached(:chef_run) { runner.converge(described_recipe) }
+      before do
+        allow(Chef::EncryptedDataBagItem).to receive(:load).with(
+          'rdiff-backup-secrets', 'secrets'
+        ).and_return(
+          key: 'secret-key'
+        )
+      end
+
+      it do
+        expect(chef_run).to create_rdiff_backup('test1')
+      end
+      # Tests lines [74:end] of the libraries/rdiff-backup.rb file
+      it do
+        expect(chef_run).to include_recipe('yum-epel')
+      end
+      # line 76
+      %w(rdiff-backup cronolog).each do |p|
+        it do
+          expect(chef_run).to install_package(p)
+        end
+      end
+      # lines 77:97, when new_resource.nrpe has been set to false
+      it do
+        expect(chef_run).to_not include_recipe('nrpe')
+      end
+      it do
+        expect(chef_run).to_not create_cookbook_file(
+          chef_run.node['nrpe']['plugin_dir'] + '/check_rdiff'
+        )
+      end
+      it do
+        expect(chef_run).to_not add_nrpe_check('check_rdiff_job_test1')
+      end
+      # line 100
+      it do
+        expect(chef_run).to create_user('rdiff-backup-server')
+      end
+      # line 101
+      it do
+        # group == owner, so group is NOT created here
+        expect(chef_run).to_not create_group('rdiff-backup-server')
+      end
+      # lines 102:108
+      it do
+        expect(chef_run).to install_sudo('rdiff-backup-server').with(
+          user: 'rdiff-backup-server',
+          group: 'rdiff-backup-server',
+          nopasswd: true,
+          commands: ['/usr/bin/sudo rdiff-backup '\
+                     '--server --restrict-read-only /']
+        )
+      end
+      # lines 109:114
+      it do
+        expect(chef_run).to create_directory('/var/rdiff-backup/locks').with(
+          owner: 'rdiff-backup-server',
+          group: 'rdiff-backup-server' || 'rdiff-backup-server',
+          mode: 0755,
+          recursive: true
+        )
+      end
+      # lines 115:119
+      it do
+        expect(chef_run).to create_directory(
+          '/home/rdiff-backup-server/.ssh'
+        ).with(
+          owner: 'rdiff-backup-server',
+          group: 'rdiff-backup-server' || 'rdiff-backup-server',
+          mode: 0700
+        )
+      end
+      # lines 120:123
+      # undefined method create_ssh_user
```

Comment:
You can remove things like line 84 but the the ``lines x:x`` is actually very helpful due to the complexity of the library. The only issue is whenever a change is made these won't be updated so it might be good to go ahead and remove them.